src/sysupdate/sysupdate-transfer.h

```diff
@@ -46,6 +46,8 @@ struct Transfer {
         PartitionChange partition_change;
 };
 
+typedef int (*TransferProgress)(const Transfer *, const Instance *, unsigned, void *);
```

Comment:
please give the last 2 params a name too. this is not just a header, this is a form of formalized documentation, too